common/vm_hot_add_remove_disk_ctrl.yml

```diff
@@ -23,6 +23,12 @@
         type: "{{ disk_controller_type }}"
         controller_number: "{{ disk_controller_number | default(omit) }}"
   register: disk_controller_facts
+  until:
```

Comment:
As before we do not use retry when setting configure, usually use retry for getting configure info.